framework/src/executioners/Transient.C

```diff
@@ -571,6 +571,7 @@ Transient::solveStep(Real input_dt)
       _console << "Picard converged!" << std::endl;
 
       _picard_converged = true;
+      _time_stepper->acceptStep();
```

Comment:
How the _____ does any of this work?!

Reply:
Going through `Transient`, `TransientMultiApp`, etc. makes my brain hurt